src/Tribe/PUE/Checker.php

```diff
@@ -1073,11 +1101,12 @@ public function get_stats() {
 			 * Filter stats and allow plugins to add their own stats
 			 * for tracking specific points of data.
 			 *
-			 * @param array                $stats          Stats gathered by PUE Checker class
-			 * @param boolean              $use_full_stats Whether to send full stats
-			 * @param \Tribe__PUE__Checker $checker        PUE Checker class object
-			 *
 			 * @since 4.5.1
+			 *
+			 * @param boolean              $use_full_stats Whether to send full stats.
+			 * @param \Tribe__PUE__Checker $checker        PUE Checker class object.
+			 *
+			 * @param array                $stats          Stats gathered by PUE Checker class.
```

Comment:
```suggestion
			 *
			 * @param array                $stats          Stats gathered by PUE Checker class.
			 * @param boolean              $use_full_stats Whether to send full stats.
			 * @param \Tribe__PUE__Checker $checker        PUE Checker class object.
			 *
```
1) in the order they are used.
2) no spaces between the lines of a list of params